projects/plugins/jetpack/modules/masterbar/masterbar/class-masterbar.php

```diff
@@ -937,7 +937,7 @@ public function add_my_sites_submenu( $wp_admin_bar ) {
 				array(
 					'url'   => $plans_url,
 					'id'    => 'wp-admin-bar-plan-badge',
-					'label' => $plan['product_name_short'],
+					'label' => ! empty( $plan['product_name_short'] ) ? $plan['product_name_short'] : esc_html__( 'Free', 'jetpack' ),
```

Comment:
Can we use `??` yet? :)

Reply:
Not yet, no. The null coalescing operator was introduced in PHP 7 and we're still supporting PHP 5.6 along with Core.